app/js/components/ArbitrationList.jsx

```diff
@@ -18,7 +22,8 @@ const ArbitrationList = (props) => (
       <CardTitle>Disputed escrows</CardTitle>
     </CardHeader>
     <CardBody>
-      {props.loading && <p>Loading...</p>}
+      {props.loading && <p><FontAwesomeIcon icon={faSpinner} className="loading"/>{props.t('arbitrationList.loading')}</p>}
+      {props.txHash && <TransactionHash txHash={props.txHash}/>}
```

Comment:
Do you think we could have a component using render props or HOC (pattern up to you) 
that abstract all of that?